app/components/computer_access.html.erb

```diff
@@ -17,9 +17,9 @@
 
   <h3 class="h2--alt"><span>Computer access</span></h3>
   <p>Places with computers you can use</p>
-  <% if places&.any? %>
+  <% if @places %>
```

Comment:
do you think its confusing to still have references to `places` in the code? should we switch these all to `partners` or is it good to denote that its only places with a fixed address this can be at (even if we dont have that enforced rn)?

Reply:
I think this is a bigger question we can tackle at a later date? Perhaps an issue dedicated to the convergence of these.

Reply:
good idea